tensorflow/lite/tools/optimize/modify_model_interface.cc

```diff
@@ -18,9 +18,10 @@ limitations under the License.
 #include <memory>
 #include <sstream>
 #include <unordered_set>
+#include <typeinfo>
```

Comment:
Why is this necessary?

Reply:
+1. 

I don't see any usage of this function eg: typeid() function

Reply:
You are right, removed it, thank you!